packages/teams-js/src/public/secondaryBrowser.ts

```diff
@@ -13,7 +13,8 @@ import { runtime } from './runtime';
  */
 export namespace secondaryBrowser {
   /**
-   * Open a URL in the secondary browser aka in-app browser
+   * Open a URL in the secondary browser, aka in-app browser. 
+   * The user will be redirected to an external browser on web and desktop
```

Comment:
This sentence only applies to web/desktop:

On web and desktop, the user will be redirected to an external browser.